packages/discord-snippet/0.1.0/package.yml

```
@@ -0,0 +1,108 @@
+# espanso configuration file
+
+# This is the default configuration file, change it as you like it
+# You can refer to the official documentation:
+# https://espanso.org/docs/
+
+# Matches are the substitution rules, when you type the "trigger" string
+# it gets replaced by the "replace" string.
+name: discord-snippet
+parent: default
+
+matches:
+    ### ----------------
+    ### Text Formatting
+    ### ----------------
+
+    # Discord Italic
+  - trigger: ":dita"
+    replace: "_$|$_"
+
+    # Discord Bold
+  - trigger: ":dbo"
+    replace: "**$|$**"
+
+  # Discord Underline
+  - trigger: ":dun"
+    replace: "__$|$__"
+
+  # Discord Bold Italic
+  - trigger: ":ditb"
+    replace: "***$|$***"
+
+  # Discord Underline Italic
+  - trigger: ":duit"
+    replace: "__*$|$*__"
+
+  # Discord Underline Bold
+  - trigger: ":dubo"
+    replace: "__**$|$**__"
+
+  # Discord Underline Bold Italic
+  - trigger: ":dubi"
+    replace: "__***$|$***__"
+
+  # Discord Strikethrough
+  - trigger: ":dst"
+    replace: "~~$|$~~"
+
+  # Discord Header 1
+  - trigger: ":dh1"
+    replace: "# $|$"
+
+  # Discord Header 2
+  - trigger: ":dh2"
+    replace: "## $|$"
+
+  # Discord Header 3
+  - trigger: ":dh3"
+    replace: "### $|$"
+
+
+  ### ----------------
+  ### Text Blocks
+  ### ----------------
+
+  # Discord Code block
+  - trigger: ":dco"
+    replace: "`$|$`"
+
+  # Discord Multi line Code Block
```

You might consider:
```yml
  - trigger: ":dmco"
    replace: |
      ```
      $|$
      ```
```
for this one, to allow multiline code between "```" on seperate lines?

---

One minor change, and a question, and then we can go ahead and merge.

---

I will do that way instead 

```suggestion
  # Discord Multi line Code Block
  - trigger: ":dmco"
     replace: "```\n $|$ \n```"
```

Because what you suggested make an Enter action on Discord and send an empty code block

---

Good point.
I would use:
```yml
  - trigger: ":dmco"
    replace: |-
      ```
      $|$
      ```
```
but yours is fine.